1.10/installing/upgrading.md

```diff
@@ -5,58 +5,59 @@ menu_order: 3.1
 
 ## Summary
 
-This document provides instructions for upgrading a DC/OS cluster from version 1.8 to 1.9. If this upgrade is performed on a supported OS with all prerequisites fulfilled, this upgrade _should_ preserve the state of running tasks on the cluster.  This document reuses portions of the [Advanced DC/OS Installation Guide][advanced-install].
+This document provides instructions for upgrading a DC/OS cluster from version 1.9 to 1.10. If this upgrade is performed on a supported OS with all prerequisites fulfilled, this upgrade _should_ preserve the state of running tasks on the cluster.  This document reuses portions of the [Advanced DC/OS Installation Guide][advanced-install].
 
 **Important:**
 
 - Review the [release notes](https://dcos.io/releases/) before upgrading DC/OS.
 - The Advanced Installation method is the _only_ recommended upgrade path for DC/OS. It is recommended that you familiarize yourself with the [Advanced DC/OS Installation Guide][advanced-install] before proceeding.
-- The [VIP features](/docs/1.10/networking/load-balancing-vips/virtual-ip-addresses/), added in DC/OS 1.8, require that ports 32768 - 65535 are open between all agent and master nodes for both TCP and UDP.
 - Virtual networks require minimum Docker version 1.11. For more information, see the [documentation](/docs/1.10/networking/virtual-networks/).
 - The DC/OS UI and APIs may be inconsistent or unavailable while masters are being upgraded. Avoid using them until all masters have been upgraded and have rejoined the cluster. You can monitor the health of a master during an upgrade by watching Exhibitor on port 8181.
 - Task history in the Mesos UI will not persist through the upgrade.
+- The latest version of Marathon-LB is required for 1.10. Before upgrading to 1.10, uninstall your existing Marathon-LB package and reinstall the updated version.
```

Comment:
The MLB packages for 1.9 and 1.10 are currently different. So to upgrade to the new MLB they will need to upgrade to 1.10 and then upgrade their MLB else `dcos package install marathon-lb` will install the older version if they are on the 1.9 cluster. 

However, this limitation is short-term, we are working on a fix that will make it backwards compatible. Not sure we need to mention this here?